### PR TITLE
Legal fixes, Yamato CI

### PR DESCRIPTION
**Pull request by kwi-dk**
- Ensure GPL license compliance.
- Add support for building on Yamato, including Stevedore artifact uploads.

### CODE REVIEW

File: .yamato/yamato-ci.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+publish:
+  name: 'Build and publish Windows 7za Stevedore artifacts, including source (GPL)'
+  agent:
+    type: Unity::VM
+    image: cds-ops/win10-katana-agent
```
Comment (richard-fine) [inline .yamato/yamato-ci.yml:5]: Why use a Katana agent image for a Yamato job?

**Comment by kwi-dk** [comment]
(Not sure how my attempt to post a comment ended up merging the PR. Mondays, I guess. Anyway, disregard that...)

File: .yamato/yamato-ci.yml (hunk 1 above)
Comment (kwi-dk) [reply .yamato/yamato-ci.yml:5]: I'm open to suggestions, if you know of another image with VS2017 C++ installed.

Comment (richard-fine) [reply .yamato/yamato-ci.yml:5]: Tried `cds-ops/win10-vs2017` ?

Comment (kwi-dk) [reply .yamato/yamato-ci.yml:5]: Right. That one has VS2017, but doesn't have Python for the script that creates the artifact files.

Comment (richard-fine) [reply .yamato/yamato-ci.yml:5]: Ah. Well, the job could `choco install python3`... or we could create a dedicated image. Or we leave it alone... the cds-ops/win10-katana-agent image is going to be retired, though (it'll still be in the catalogue but won't be maintained).

Comment (kwi-dk) [reply .yamato/yamato-ci.yml:5]: I'm not too concerned about long-term support for the Bokken image. Given the interval between us building 7za so far, any image named `*-vs2017` will almost certainly be out of date by then, anyway.

So I'm leaning towards just pushing this, and we can look at available images the next time we need to rebuild 7za, if `cds-ops/win10-katana-agent` is obsolete then.

Comment (richard-fine) [reply .yamato/yamato-ci.yml:5]: Alrighty.

**Pull request merged**